composer.json

```diff
@@ -30,7 +30,7 @@
         "symfony/dependency-injection": "~2.7|~3.0|~4.0",
         "doctrine/dbal": "^2.5.12",
         "jdorn/sql-formatter": "^1.2.16",
-        "symfony/doctrine-bridge": "~2.7|~3.0|~4.0",
+        "symfony/doctrine-bridge": "^4.2",
```

Comment:
We'll be able to use feature detection before merging.

Reply:
aren't we already able to do it ?

Reply:
Not sure if there is a new class yet.

Reply:
done